pkg/controller/controller_test.go

```diff
@@ -477,13 +477,18 @@ func provisionFromPVCCapabilities() (rpc.PluginCapabilitySet, rpc.ControllerCapa
 		}
 }
 
-// Minimal PVC required for tests to function
-func createFakePVC(requestBytes int64) *v1.PersistentVolumeClaim {
+// createFakeNamedPVC creates PVC for tests, should be extended with new args when needed
+func createFakeNamedPVC(requestBytes int64, name string, userAnnotations ...struct{ Key, Value string }) *v1.PersistentVolumeClaim {
```

Comment:
Yes, the current signature makes `userAnnotations` argument optional. If we change it to map it will require to add an empty map to each place where this function is invoked now.

Reply:
Seems like not too many call sites? Alternatively, since you only use `userAnnotations` in one place, you could make it a different helper function with a longer, more specific name.

The most comprehensive solution would be to adopt the standard "function options" pattern, but that may be overkill.

Reply:
Hm, I agree with you, a different helper function is the better way to handle this.

Reply:
Fixed, it is `map[string]string` now.